examples/data/exercise-errors.rb

```diff
@@ -15,7 +15,7 @@
 errors_api = MuxRuby::ErrorsApi.new
 
 # ========== list-errors ==========
-errors = errors_api.list_errors(:filters => ['browser:Chrome'], :timeframe => ['7:days'])
+errors = errors_api.list_errors(:filters => ['browser:Safari'], :timeframe => ['7:days'])
```

Comment:
This change is to support the new synthesized views, which are identified as Safari.